### ISSUE TITLE
Cursor placement in Fix Character Values is not working right

### ISSUE THREAD

**Issue by shanaqui** [HabitRPG/habitica#11400]
This one is difficult to phrase! In Settings on the web, in Chrome, if I click into the various fields for Fix Character Values, instead of putting the caret where you click, it doesn't align properly. 

For example:

![image](https://user-images.githubusercontent.com/8144640/66164045-ace13a00-e629-11e9-94a8-5bafb568d5f4.png)

The red arrow shows where the caret showed up, but the blue arrow is showing where I actually clicked.

Confirmed on another computer too.

**Comment by LikeLakers2** [comment]
Hey there! This appears to be the fault of Chrome, not some Habitica script (otherwise it would have been noticed much sooner). I am able to replicate this caret position issue in Chrome within an online HTML editor. (note: I've not tested it on other browsers)

Here is the minimal HTML and CSS needed to be able to replicate this within Chrome:

```html
<style>
input.form-control {
    padding: 10px 16px; /* The higher the second number, the worse the issue is */
}
</style>

<input type="number" class="form-control" value="28.963558655357772">
```

**Comment by LikeLakers2** [comment]
Woop, I meant to add this bit to my above comment... guess I forgot. And it's been a couple days, so I'm making this a new comment so it'll be seen.

If this needs reporting to the Chromium project, would someone else be able to do it? I'm a little afraid to do it myself since I'm not really too experienced with HTML and CSS, and I don't want to give them a barebones bug report as a result of that. Sorry if that annoys anyone. >_<

**Comment by SuperMaltese** [comment]
I'd like to ask if this is still an issue? I tried the above minimal example and Habitica Fix Values in Chrome version 78.0.3904.70 and didn't see any problem.
Perhaps they fixed it in the most recent (Oct 22) patch?

**Comment by shanaqui** [comment]
It's still an issue for me. :)

**Comment by LikeLakers2** [comment]
It appears to have been fixed for me too, after updating to Chrome 78.0.3904.70.

@shanaqui Make sure Chrome is up-to-date. Follow the instructions [from here](https://support.google.com/chrome/answer/95414) under the section "Check for an update & the current browser version". If an update is available, let it download, then relaunch Chrome, then see if you can still replicate it.

**Comment by shanaqui** [comment]
Hmm, I thought my Chrome was up to date, but looks like it still needed to relaunch. Yep, this is fixed!
